src/xdocs/index.xml.vm

```diff
@@ -97,8 +97,27 @@
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
       <p>
-        Checkstyle is not using Semantic Versioning
-        (<a href="https://github.com/checkstyle/checkstyle/issues/3709">reasons in issue #3709</a>)
+        Checkstyle is following
+        <a href="https://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/">
+          Hybrid Romantic and Semantic Versioning</a>: This is in the form of
```

Comment:
We do not use "Hybrid Romantic and Semantic" version scheme.

From the article: 
>Hybrid Romantic & Semantic Versioning
There are many projects that maintain two different version numbers simultaneously: a romantic version for marketing purposes and a semantic version for developer purposes.

An example of this is Java 8. Java 8 is the "Romantic" version number, and 1.8 is the "Semantic" version number. Checkstyle does not use two different version numbers for the same release.

Reply:
i removed link, we create our own term.
I keep Hybrid as in my mind it is synonym of combination. We do combine two versioning.